src/js/components/Button/doc.js

```diff
@@ -45,7 +45,10 @@ export const doc = Button => {
       `The DOM tag or react component to use for the element.`,
     ),
     color: colorPropType.description(
-      'Fill color for primary, label color for plain, border color otherwise.',
+      `Fill color for primary, label color for plain, border color otherwise.
+       If button.default is defined in the theme, the color prop will
+       affect the background color of the button, regardless of which
```

Comment:
```suggestion
       affect the background color of the primary button, regardless of which
```
Should we be more accurate and point out that this color will fill the background only when the button is primary? also, how would it affect non-primary button?

Reply:
This will fill the background color if primary, secondary for default it will only change the text color I can add that